README.md

```diff
@@ -20,6 +20,15 @@ To use River-Guide, you'll need to have:
 
 ## Installation
 
+You can use the [pre-built binaries](https://github.com/frgrisk/river-guide/releases).
+
+```bash
+curl -L https://github.com/frgrisk/river-guide/releases/download/v0.2.0/river-guide-linux-amd64 --output /opt/river-guide
+chmod 755 /opt/river-guide
```

Comment:
Can we update this to be a bit more robust so that we don't have to update for each release?

```suggestion
if [[ $(uname -m) == aarch64 ]]; then
   GOARCH=arm64
else
   GOARCH=amd64
fi
curl -LO https://github.com/frgrisk/river-guide/releases/latest/download/river-guide-linux-$GOARCH
chmod +x river-guide-linux-*
sudo cp river-guide-linux-* /usr/local/bin/river-guide
```